src/FretboardSVG.module.css

```
@@ -0,0 +1,284 @@
+  transition: opacity 0.15s ease-out;
+}
+
+:global(.fretboard-note).hidden {
+  display: none;
+}
+
+:global(.fretboard-note):focus {
+  outline: none;
+}
+
+:global(.fretboard-note):focus-visible circle,
```

Stylelint (17.7.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)